docs/guides/testing.md

```diff
@@ -57,10 +57,22 @@ this means your application logic does not need to be changed or mocked when wri
 
 The mock provided below will enable the relevant test runner to reset the zustand stores after each test.
```

Comment:
This paragraph indicates that the next thing will be a mock of the store, but there is an unexpected _shared code_ section.

It isn't clear from the context, but is the _shared code_ shared between production and test code? If so, you can write a paragraph to explain the purpose. You can also drop the _shared code_ heading and keep everything in one section with a note that _shared code_ is the recommended way of dealing with Zustand in tests.

Reply:
@sewera the shared code part as I mentioned in the docs is just for testing purposes due to duplication of code on both test cases w/o context (create and createStore + context)

Reply:
@sewera lmk if that makes sense, btw I can add a short description if that's necessary

Reply:
If you could add it, I think it will read better :)